features/formatters.feature

```diff
@@ -1,17 +1,17 @@
 Feature: Formatters
 
-  Scenario: rejected pickle
+  Scenario: rejected pickle from Scenario
     Given a file named "features/a.feature" with:
       """
       Feature: a feature
         Scenario: a scenario
           Given a step
       """
     When I run cucumber-js with all formatters and `--tags @a`
-    Then the "message" formatter output matches the fixture "formatters/rejected-pickle.message.json"
+    Then the "message" formatter output matches the fixture "formatters/rejected-pickle-scenario.message.json"
```

Comment:
can we also revert the changes to this scenario and the failed scenario so that we only specify the difference for scenario vs rule for the passed scenarios?